__tests__/app.test.js

```diff
@@ -252,3 +252,42 @@ describe("ERROR: POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("Patch /api/articles/:article_id", () => {
+  test("201: should respond with the updated article", () => {
+    return request(app)
+      .patch(`/api/articles/1`)
+      .expect(201)
+      .send({
+        inc_votes: 10,
+      })
+      .then(({ body: { article } }) => {
+        expect(article.votes).toBe(110);
+      });
+  });
+});
+
+describe("ERROR: Patch /api/articles/:article_id", () => {
+  test("400: responds with an error message when passed a bad request", () => {
+    const article_id = "invalid_type";
+    return request(app)
+      .patch(`/api/articles/${article_id}`)
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Bad Request");
+      });
+  });
+
+  test("404: responds with an error message when changing a article which is non-existent", () => {
+    const article_id = 999;
+    return request(app)
+      .patch(`/api/articles/${article_id}`)
+      .send({
+        inc_votes: 10,
+      })
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Not Found");
+      });
+  });
+});
```

Comment:
Please add a test for the following:

1. 201 that will ignore any additional properties 👍 
2. 400, missing required field(s), e.g. no username or body properties
3. 404, username does not exist